scripts/jest/setupTests.js

```diff
@@ -231,15 +254,19 @@ if (process.env.REACT_CLASS_EQUIVALENCE_TEST) {
       ) {
         await maybePromise;
       }
+      // Flush unexpected console calls inside the test itself, instead of in
+      // `afterEach` like we normally do. `afterEach` is too late because if it
+      // throws, we won't have captured it.
+      flushAllUnexpectedConsoleCalls();
     } catch (error) {
       // Failed as expected
+      resetAllUnexpectedConsoleCalls();
       return;
     }
-    throw Error(
-      'Tests marked experimental are expected to fail, but this one passed.'
-    );
+    throw Error(errorMsg);
   };
 
+  // TODO: Deprecate these helpers in favor of @gate pragma
```

Comment:
Why? What value does a custom pragma offer for iterating on a local focused test?

Reply:
Oh I meant it.experimental, not the focus helpers. Definitely not removing those!

Reply:
Ooh, I misunderstood that comment!

That makes sense then.